arangodb-net-standard/SerializerException.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace ArangoDBNetStandard
+{
+    [Serializable]
+    public class SerializerException : Exception
```

Comment:
What do you think of naming this `SerializationException` instead of `SerializerException`? I think "serialization" makes it more related to the serialization process in general rather than a specific "serializer".

Reply:
done